src/Commands/ImportDump.php

```diff
@@ -87,6 +89,7 @@ public function handle()
 
         $options                     = [];
```

Comment:
Why an options array just to pass it in line 243? I would prefer having the array in line 243 to also see the possible options.